ios/Classes/strategies/AssistantPermissionStrategy.m

```diff
@@ -0,0 +1,81 @@
+//
+// Created by Basit Mustafa (24601@GitHub) on 2019-09-13.
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+//
+
+#import "AssistantPermissionStrategy.h"
+
+
+@implementation AssistantPermissionStrategy
+
+- (PermissionStatus)checkPermissionStatus:(PermissionGroup)permission {
+    if (permission == PermissionGroupAssistant) {
+        if (@available(iOS 10.0, *)) {
+            return [AssistantPermissionStrategy permissionStatus];
+        } else {
+            // Fallback on earlier versions
```

Comment:
Remove this empty block, since SiriKit is not available on earlier versions it makes no sense to keep it here.